### ISSUE TITLE
new release do not work with julia 0.5

### ISSUE THREAD

**Issue by xiuliren** [JuliaImages/ImageCore.jl#43]
I just updated the packages, and Images do not work with my julia 0.5. I noticed that the code is compatible with julia 0.6, but this release should not be updated within julia 0.5!

```
using Images
INFO: Recompiling stale cache file /usr/people/jingpeng/.julia/lib/v0.5/ImageMetadata.ji for module ImageMetadata.
ERROR: LoadError: LoadError: syntax: extra token "ChannelView" after end of expression
 in include_from_node1(::String) at ./loading.jl:488 (repeats 2 times)
 in macro expansion; at ./none:2 [inlined]
 in anonymous at ./<missing>:?
 in eval(::Module, ::Any) at ./boot.jl:234
 in process_options(::Base.JLOptions) at ./client.jl:239
 in _start() at ./client.jl:318
while loading /usr/people/jingpeng/.julia/v0.5/ImageCore/src/colorchannels.jl, in expression starting on line 29
while loading /usr/people/jingpeng/.julia/v0.5/ImageCore/src/ImageCore.jl, in expression starting on line 62
ERROR: LoadError: Failed to precompile ImageCore to /usr/people/jingpeng/.julia/lib/v0.5/ImageCore.ji.
```

**Comment by xiuliren** [comment]
after removing the package and reinstall, it works now.

**Comment by timholy** [comment]
New versions of ImageCore will require 0.6, so best to stick with released versions if you're using 0.5. My guess is that you were using `master`.
